gradio/blocks.py

```diff
@@ -1184,32 +1185,32 @@ def clear(self):
     def queue(
         self,
         concurrency_count: int = 1,
-        status_update_rate: float | str = "auto",
-        client_position_to_load_data: int = 30,
+        status_update_rate: float | Literal["auto"] = "auto",
+        client_position_to_load_data: int | None = None,
         default_enabled: bool = True,
         api_open: bool = True,
-        max_size: Optional[int] = None,
+        max_size: int | None = None,
     ):
         """
         You can control the rate of processed requests by creating a queue. This will allow you to set the number of requests to be processed at one time, and will let users know their position in the queue.
         Parameters:
-            concurrency_count: Number of worker threads that will be processing requests concurrently.
+            concurrency_count: Number of worker threads that will be processing requests from the queue concurrently. Increasing this number will increase the rate at which requests are processed, but will also increase the memory usage of the queue.
             status_update_rate: If "auto", Queue will send status estimations to all clients whenever a job is finished. Otherwise Queue will send status at regular intervals set by this parameter as the number of seconds.
-            client_position_to_load_data: Once a client's position in Queue is less that this value, the Queue will collect the input data from the client. You may make this smaller if clients can send large volumes of data, such as video, since the queued data is stored in memory.
```

Comment:
client_position_to_load_data is not documented in the docstring


<img width="1087" alt="Screen Shot 2022-12-15 at 7 13 56 PM" src="https://user-images.githubusercontent.com/9021060/207897280-b09141a7-3d62-448e-af67-8a75505e4a68.png">

Reply:
Fixed now